plugins/inputs/win_perf_counters/README.md

```diff
@@ -384,6 +389,30 @@ if any of the combinations of ObjectName/Instances/Counters are invalid.
     #IncludeTotal=false #Set to true to include _Total instance when querying for all (*).
 ```
 
+### Multiple plugins with unique tags
```

Comment:
Don't add this, creates a bad precedent for documenting this general option everywhere.